app/mailers/hypertension_fast_track_pilot_mailer.rb

```diff
@@ -5,22 +5,21 @@ def build(submission)
     @submission = submission
     @disability = submission.form.dig('form526', 'form526', 'disabilities')&.first
     @disability_code = @disability['diagnosticCode'] if @disability
-    @rrd_claim_processed = RapidReadyForDecision::Form526BaseJob.rrd_claim_processed?(submission)
-    @bp_readings_count = submission.form.dig('rrd_med_stats', 'bp_readings_count') || 'N/A'
+    @bp_readings_count = submission.form.dig('rrd_metadata', 'med_stats', 'bp_readings_count') || 'N/A'
 
-    subject =
-      if @rrd_claim_processed
-        'RRD claim - Processed'
-      else
-        'RRD claim - Insufficient Data'
-      end
-    subject += " - #{@disability_code}"
+    @description = if RapidReadyForDecision::Form526BaseJob.rrd_claim_processed?(submission)
```

Comment:
I wonder if there could be a method on Form526BaseJob or something that could determine `rrd_status` and return symbols? Not necessary right this second but might be useful in the future, in case this needs to be centralized somewhere.